net-analyzer/zeek/zeek-3.0.1.ebuild

```diff
@@ -0,0 +1,125 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7,8} )
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="The Zeek Network Security Monitor"
+HOMEPAGE="https://www.zeek.org"
+SRC_URI="https://www.zeek.org/downloads/${P}.tar.gz"
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="curl debug geoip2 ipsumdump ipv6 jemalloc kerberos +python sendmail \
+	static-libs tcmalloc +tools +zeekctl"
+
+RDEPEND=">=sys-libs/glibc-2.10
+	dev-libs/actor-framework:0=
+	dev-libs/openssl:0=
+	net-libs/libpcap
+	sys-libs/zlib:0=
+	curl? ( net-misc/curl )
+	geoip2? ( dev-libs/libmaxminddb )
+	ipsumdump? ( net-analyzer/ipsumdump[ipv6?] )
+	jemalloc? ( dev-libs/jemalloc:0= )
+	kerberos? ( virtual/krb5 )
+	python? ( ${PYTHON_DEPS}
+		dev-python/pybind11[${PYTHON_USEDEP}] )
+	sendmail? ( virtual/mta )
+	tcmalloc? ( dev-util/google-perftools )"
+
+DEPEND="${RDEPEND}"
+
+BDEPEND=">=dev-lang/swig-3.0
+	>=dev-util/cmake-2.8.12
```

Comment:
Cmake should come from cmake eclass, gcc is implicit, AFAIK flex too, you could drop them from here

Reply:
Done